src/models/target/index.ts

```diff
@@ -23,16 +22,21 @@ export interface IStandards {
   stdDesc: string;
 }
 
+export interface IEvidence {
```

Comment:
Good thinking adding an interface for this. we may have to update the types on the client side too so the client-side team can take advantage of them.